lib/auth/api.go

```diff
@@ -786,6 +787,18 @@ type ReadOktaAccessPoint interface {
 	// GetUsers returns a list of users with the cluster
 	GetUsers(ctx context.Context, withSecrets bool) ([]types.User, error)
 
+	// ListAccessLists returns a paginated list of access lists.
+	ListAccessLists(context.Context, int, string) ([]*accesslist.AccessList, string, error)
```

Comment:
Naming these arguments would help to immediately understand their purpose.

Reply:
+1